gitregostore/gitstoreutils.go

```diff
@@ -33,6 +34,7 @@ const (
 var controlIDRegexCompiled *regexp.Regexp = nil
 
 var storeSetterMapping = map[string]storeSetter{
+	// attackTracksJsonFileName:          (*GitRegoStore).setAttackTracks,	// TODO: Uncomment when attack tracks are ready
```

Comment:
Should be commented out once https://github.com/kubescape/regolibrary/pull/94 is merged